puppet/modules/slave/manifests/init.pp

```diff
@@ -303,15 +297,6 @@
     }
   }
 
-  # Cleanup Jenkins Xvfb processes from aborted builds after a day
-  file { '/etc/cron.daily/xvfb_cleaner':
-    ensure  => file,
```

Comment:
Can you change this to `ensure => absent` so we actually remove it?

Reply:
updated